design/ceph/object/ceph-sse-s3.md

```diff
@@ -0,0 +1,42 @@
+---
+# AWS server side encryption SSE-S3 support for RGW
+target-version: release-1.10
+---
+
+# Feature Name
+AWS server side encryption SSE-S3 support for RGW
+
+## Summary
+The S3 protocol over three different types of [server side encryption](https://docs.aws.amazon.com/AmazonS3/latest/userguide/serv-side-encryption.html): SSE-C, SSE-KMS and SSE-S3. For the last two RGW server need to configure with external services such as [vault](https://www.vaultproject.io/). Currently Rook configure RGW with `SSE-KMS` options, so that S3 request with `sse:kms` header will be handled properly. Recently the support for handling the `sse:s3` added to RGW with another set of options, so Rook needs to provide option to configure RGW similar to `SSE-KMS`.
```

Comment:
"The S3 protocol over three different types..." doesn't make sense. Please revise.